### ISSUE TITLE
Filling form results in undefined method `*' for nil:NilClass

### ISSUE THREAD

**Issue by thomasbaustert** [gettalong/hexapdf#244]
Hi and thanks for hexapdf!
I tried to fill out the following PDF template which results in `undefined method `*' for nil:NilClass`.
The `cap_height` is `nil`.
Other PDF templates work fine so far but this one failed.
All templates are created with Adobe.

```ruby
require 'hexapdf'

values = {
  'Tagesdatum' => Date.today.strftime('%d.%m.%Y')
}.freeze

doc = HexaPDF::Document.open('vorlage-test.pdf')
doc.acro_form.each_field do |field|
  value = values.fetch(field.field_name)

  field.field_value = value.to_s
  field.flag(:read_only)
end
```

[vorlage-test.pdf](https://github.com/gettalong/.github/files/11274018/vorlage-test.pdf)

**Comment by gettalong** [comment]
I have found the problem and a solution. Will incorporate the fix into a release due tomorrow.

**Comment by thomasbaustert** [comment]
Thanks for the quick feedback!

**Comment by gettalong** [comment]
I just released 0.32.1 with the fix.
